requirements.txt

```diff
@@ -1,4 +1,4 @@
--e git+git://github.com/fredj/cssmin.git@master#egg=cssmin
+git+https://github.com/fredj/cssmin.git
```

Comment:
Reading the [documentation](https://pip.pypa.io/en/stable/reference/pip_install/#git) it seems for pip #egg=cssmin is required so can't it be kept? Why would poetry not understand something pip makes up?

Reply:
I think the point was removing the `-e` which poetry can't handle, not sure why the egg spec is being removed.

Reply:
Just tested it and poetry can't handle that @master either which is rather sad

Reply:
I would raise an issue with `pip` for deprecating ` #egg=cssmin` suffix that looks like a hack.

Reply:
It is absolutely not. And it is already deprecated as eggs themselves are deprecated.
https://pip.pypa.io/en/stable/reference/pip_install/#vcs-support

Reply:
Absolutely not what?

Reply:
It is absolutely not a hack.

Reply:
Press :smile: if `#egg` is a hack, and :tada: if it is not. )

Reply:
It's fine to argue if this is or is not a hack, but please do not go open issues for features very deliberately documented. This is open source and I assure you all the pip maintainers, as is usual on open source projects, are busy, you will only be wasting their already limited time.

Reply:
I am using `poetry`. So, what is the consensus Are you going to merge this? )